app/src/main/java/com/github/se/polyfit/viewmodel/dailyRecap/DailyRecapViewModel.kt

```diff
@@ -23,11 +26,15 @@ class DailyRecapViewModel @Inject constructor(private val mealRepository: MealRe
   private val _isFetching: MutableStateFlow<Boolean> = MutableStateFlow(false)
   val isFetching: StateFlow<Boolean> = _isFetching
 
+  var date: LocalDate by mutableStateOf(LocalDate.now())
+
   init {
-    getMealsOnDate(LocalDate.now())
+    getMealsOnDate()
   }
 
-  fun getMealsOnDate(date: LocalDate) {
+  fun getMealsOnDate(newDate: LocalDate = date) {
+    date = newDate
+
```

Comment:
What is the purpose to put the old date as by default argument to the new date?

Reply:
So that it saves the date state upon returning from edit, otherwise we can not keep them on the same date when they return from edits